WireUI/Sources/WireReusableUIComponents/UIViewController+NavigationBarTitle.swift

```diff
@@ -0,0 +1,63 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import UIKit
+import WireDesign
+
+public extension UIViewController {
+
+    func setupNavigationBarTitle(with title: String) {
+        navigationItem.title = title.capitalized
+
+        let titleTextAttributes: [NSAttributedString.Key: Any] = [
+            .foregroundColor: SemanticColors.Label.textDefault,
+            .font: UIFont.systemFont(ofSize: 17, weight: .semibold)
+        ]
+
+        navigationController?.navigationBar.titleTextAttributes = titleTextAttributes
+
+        navigationItem.titleView?.isAccessibilityElement = true
+        navigationItem.titleView?.accessibilityTraits = .header
+        navigationItem.titleView?.accessibilityLabel = navigationItem.title
+
+        if let titleView = navigationItem.titleView {
+            titleView.showsLargeContentViewer = true
+            titleView.largeContentTitle = navigationItem.title
+        }
```

Comment:
Why is this if let here only for the last 2 lines? Can't it be like:
```swift
navigationItem.titleView?.showsLargeContentViewer = true
navigationItem.titleView?.largeContentTitle = navigationItem.title
```

Reply:
Done: https://github.com/wireapp/wire-ios/commit/812da547eb79887c3b53450dca6e59e40233710f